dll/win32/browseui/CAutoComplete.cpp

```diff
@@ -53,7 +53,7 @@ static const PREFIX_INFO s_prefixes[] =
     { L"www.", 4 },
 };
 
-static inline BOOL DropPrefix(const CStringW& str, CStringW& strBody)
+static BOOL DropPrefix(const CStringW& str, CStringW& strBody)
```

Comment:
why did you remove inline?

Reply:
I thought it is too large to be embedded.